x-pack/legacy/plugins/snapshot_restore/public/app/lib/authorization/components/not_authorized_section.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+interface Props {
+  sectionName: string;
+  privilegeType: string;
+  missingPrivileges: string[];
+  FormattedMessage: typeof ReactIntl.FormattedMessage;
+}
+
+export const NotAuthorizedSection = ({
+  sectionName,
+  privilegeType,
+  missingPrivileges,
+  FormattedMessage,
+}: Props) => (
+  <EuiEmptyPrompt
+    iconType="securityApp"
+    title={
+      <h2>
+        <FormattedMessage
+          id="xpack.snapshotRestore.restoreList.deniedPermissionTitle"
+          defaultMessage="You're missing {privilegeType} privileges"
+          values={{
+            privilegeType,
```

Comment:
this will run into localization problems since `cluster`/`index` won't be part of the translation. see my next comment